xlators/cluster/dht/src/dht-common.h

```
@@ -762,6 +762,23 @@ typedef struct dht_fd_ctx {
         }                                                                      \
     } while (0)
 
+#define DHT_MARK_RENAME_FOP(xattr)                                             \
```

Why a macro instead of a function ?

---

This should be a function, not a macro. Using a function you can return an error in case xattr allocation or dict_set_str fails.